_projects/2022/100386498/100386498.Rmd

```diff
@@ -0,0 +1,390 @@
+
+```{r fig.width=15,fig.height=10,fig.align = "center"}
+
+p <- p + theme(legend.text=element_text(size=10),
+        legend.position = c(0.095,0.91), 
+        legend.title = element_blank(),  
+        legend.key = element_blank())
+p
+```
+
+### Background
+The last touch to replicate the original chart is to color the background in light grey with `plot.background`. To do this, I first use `plot.margin` to control the margins around the plot. This code sets top margin to 1 cm, the right margin to 4 cm, the bottom margin to 1 cm, and the left margin to 4 cm. Despite setting the caption to the left of the plot, I could not figure put how to align it with the title and the subtitle.
+
+```{r fig.width=15,fig.height=10,fig.align = "center"}
+ 
+p <- p + theme(plot.margin = unit(c(1,4,1,4),"cm"),
+          plot.background = element_rect(fill="#f8f8f8"),
+          plot.title = element_markdown (hjust =-0.35),
+          plot.subtitle = element_markdown(hjust=-0.25),
+          plot.caption.position = "plot",
+          plot.caption = element_markdown(hjust= -0.25))   
+p
+```
+
+## Final chart
+
+The World Bank's chart is trying to represent the relationship between the HFCE and the national poverty line to showcase how richer countries have higher national poverty lines. The visual channels I will focus on to build an alternative chart are the xy position and the hue.
+
+To represent this relationship it is important that the dots of the scatterplot inform us about the values of each country and the global pattern. One possible way to deal with overplotting faceting: the same plot is repeated, each time highlighting another group. A nice way to keep the global and the between groups comparisons is to plot the non-group dots in the background and in light grey (general context data).
+
+Moreover, I have decided to keep the logarithmic transformation because it helps the visualization of the data. It also helps to the representation of the information that the axes are not shown in logarithmic scale, and are rather displayed in the original scale.
+
+As in the original graph it was a bit confusing to know precisely which dot the label was referring to, by now having four facets, there is more space to specify each country. To do this I used `library(ggrepel)`.
+
+In addition, to enhance the colors of the dots I used the color scheme by [Carto](https://carto.com/carto-colors/) to make more visible the different groups.
+
+```{r fig.show='hide'}
+p1 <- ggplot(qntile, aes(HFCE, national_povt_line, color=country_class)) + 
+  geom_point(data = transform(qntile,
+     country_class = NULL), colour = "grey80") + 
+  geom_point(alpha=0.7) +
+  ggrepel::geom_text_repel(aes(label=country)) +
+  scale_color_manual(values = c("#5F4690","#38A6A5", "#73AF48", "#E17C05")) +
+  facet_grid(~country_class, scales = "free") + 
+  theme_bw()
+```
+
+```{r echo=FALSE, fig.width=15,fig.height=12,fig.align = "center"}
```

Comment:
Please do not hide any chunk. Remove the `echo=FALSE`. In fact, you could merge this chunk with the previous one.

Reply:
Additionally, I think this plot would look nicer with something like `fig.asp="0.3"`  (after removing the current `fig.height`) and `layout="l-page"` or even `layout="l-screen"`. You can play with those values.